Dockerfile

```diff
@@ -3,7 +3,7 @@ MAINTAINER CyberArk Software, Ltd.
 
 ENV GOOS=linux \
     GOARCH=amd64 \
-    CGO_ENABLED=0
+    CGO_ENABLED=1
```

Comment:
why do we need `cgo_enabled`? are you sure it is required?

Reply:
I am not sure about this one. Other projects used it compile source code in Go and C

Reply:
do you think we need this offering here?

Reply:
please don't enable cgo if we don't have to. if the build passes without it then leave it like that.